samples/command-line-terminal.yaml

```diff
@@ -0,0 +1,31 @@
+# It's just an example of workspace configuration OpenShift Console creates
+# when user requests terminal
+apiVersion: workspace.che.eclipse.org/v1alpha1
+kind: Workspace
+metadata:
+  name: command-line-terminal
+  annotations:
+    # it's important to make workspace immutable to make sure nobody with right access
+    # won't set custom editor to steal creator's token
+    org.eclipse.che.workspace/immutable: "true"
+  labels:
+    # it's a label OpenShift console uses a flag to mark terminal's workspaces
+    console.openshift.io/cloudshell: "true"
+spec:
+  started: true
+  devfile:
+    apiVersion: 1.0.0
+    metadata:
+      name: command-line-terminal
+    components:
+      - alias: command-line-terminal
+        type: cheEditor
+        id: che-incubator/command-line-terminal/4.5.0
```

Comment:
should this be a moving tag `:4.5` or do you plan to release a new release for every OCP 4.5.x release too?

Reply:
Good question :+1:
I think it's a safer option to release command-line-terminal version specially for the concrete OpenShift version. I can imagine a situation where OpenShift 4.5.0 and OpenShift 4.5.1 possible need different APIs from command-line-terminal.

Or instead, we're able not stick to OpenShift version and do the right Major.minor.patch versioning where we make sure that we do not provide breaking changes.
Then OpenShift Console will get bug fixes without any changes in their code base, and version in OpenShift Console will be changed only after OpenShift Console Frontend adaptation.